README.md

```diff
@@ -77,13 +79,97 @@ In the sub-folders for plots that intersected with the orthomosaic, there will b
 The CSV files both contain much of the same information.
 The file with "geostreams" in its name can be uploaded to TERRAREF's Geostreams database.  
 
+### Canopy Cover: OpenDroneMap and plot boundaries <a name="opendm_can_shp" />
+The following steps are used when wanting to use OpenDroneMap (ODM) to create the Orthomosaic image that's then used to create the canopy cover values.
+As with the [previous example](#om_can_shp) we will be listing the steps and then providing an example.
+
+_NOTE_: the SciF Docker image uses _Docker siblings_ to run the OpenDroneMap application.
+If you are not comfortable with this approach, you shouldn't use this workflow.
+
+1. Create two named Docker volumes to use for processing data; one for input files and one for output files - the same volume can be used for both if desired
+2. Copy the source drone images into a folder
+3. If using a [shapefile](#shapefile) or [GeoJSON](#geojson) file, copy those into the same folder as the drone images
+4. Copy the experiment metadata file into the same folder as the drone images
+5. Copy the folder contents of the drone images folder that was just prepared onto the input named volume
+6. Create another folder for the output folders and files
+7. Run the docker container's `odm_workflow` app specifying  either the name of the shapefile or geojson file, or the URL of they [BETYdb](#betydb) instance to query for plot boundaries, and the two named volumes
+8. Copy the resulting files off the output named volume to the local folder
+9. Clean up the named volumes
+
+#### For example: <a name="opendm_can_shp_example" />
+
+In this example we're going to assume that we're using a shapefile named `plot_shapes.shp`, and that we have our drone images in a folder named `~/IMG`.
```

Comment:
also assuming an orthomosaic tif file? 

As a user I just want to be able to copy-paste stuff and get it done. can you add instructions for downloading these (e.g. using `curl`)? and to not rely on the user's home directory (e.g. not putting new files in ~/, perhaps doing everything relative to /tmp/ or `mkdir drone-demo && cd drone-demo`?

Reply:
sorry, I meant the input files

Reply:
The folder names have been changes to start with `/` and not `~/`. A sample set of data is available on Google Drive and the link included in the documentation.

Reply:
> A sample set of data is available on Google Drive and the link included in the documentation.

Is it possible to download this from the command line from where it is on google drive, or is there a folder on the data store that we use as an FTP server? For a tutorial it would be nice if everything can be automated at the command line.

Reply:
@dlebauer Not FTP, but the IRods tool set can be installed and a file downloaded from there. Alternatively, there could be a dropbox account where files can be retrieved using `curl`. To download from Google Drive via command line, a script needs to be used or Google command line tools installed. Is there a particular reason for not wanting to use a browser to download files - to only operate in a headless (no UI) environment?

Reply:
mostly so that the automation is clear; not to 'only operate' w/o UI, but so that this is possible. curl also has the advantage of putting the data where you want it w/o having to navigate the filesystem UI when either downloading or moving the downloaded data over. If I am on an Rstudio server for example, it adds the extra step of downloading to my local desktop and then uploading to Rstudio. 

Another reason is that if we have sample data on FTP it would make it possible to convert the README examples into an integration test. 

Definitely shouldn't need to add the irods toolbox - I am pretty sure you can curl data from CyVerse data store - isn't that how the BETYdb dump works?

Reply:
Welsch uses irods commands to put the files up onto CyVerse. Curl can be used but isn't due to security risks associated with having to specify credentials in cron jobs

Reply:
I have updated the README with how to use the `curl` command to download from iRods and decompress the archive

Reply:
The folder name changes to `/` have been lost. I'm adding them back in again.